src/components/data-exploration/differential-expression-tool/DiffExprCompute.jsx

```diff
@@ -30,7 +30,7 @@ const DiffExprCompute = (props) => {
   const dispatch = useDispatch();
   const { properties, hierarchy } = useSelector(getCellSets());
   const [isFormValid, setIsFormValid] = useState(false);
-  const [numSamples, setNumSamples] = useState(1)
+  const [numSamples, setNumSamples] = useState(0);
```

Comment:
Why was this changed? (and do you know why was it 1 before?)

Reply:
It was set to 1 before because we assume that there will always be at least 1 sample. That is a correct assumption of course, but we have some behaviors which will trigger specifically when the number of samples is 1, even while the cell set is still loading. After the cell set is loaded, the number of samples might change. This means we'll always trigger the behaviour specific for 1 sample, even though in reality there are more than 1 samples. So, I think the better option is to set the initial number of cellsets to 0, which will then be corrected after the cell sets load.

Reply:
nice